es/11_0_Potenciando_Bloqueos_de_Tiempo_con_Bitcoin_Scripts.md

```diff
@@ -0,0 +1,23 @@
+# Capitulo 11: Potenciando Bloqueos de Tiempo con Bitcoin Scripts
+
+La mejora que introdujo `nLockTime` en [§8.1](08_1_Enviando_una_Transaccion_con_Bloqueo_de_Tiempo.md) fue solo el comienzo del principio de los bloqueos de tiempo. Cuando comienzas a escribir Bitcoin Scripts, dispones de dos opcodes de bloqueo de tiempo.
```

Comment:
All text should use formal way instead informal address as "Tú".
So this line could be:
"Cuando comienza a escribir scritps de Bitcoin,  dispone de dos opcodes de bloqueo de tiempo"